docs/guides/configure-acl-plugin.md

```diff
@@ -0,0 +1,694 @@
+  name: demo-get
+  annotations:
+    konghq.com/strip-path: "false"
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /get
+        backend:
+          serviceName: httpbin
+          servicePort: 80
+' | kubectl apply -f -
+
+$ echo '
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: demo-post
+  annotations:
+    konghq.com/strip-path: "false"
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /post
+        backend:
+          serviceName: httpbin
+          servicePort: 80
+' | kubectl apply -f -
+```
+
+Test the Ingress rules:
+
+```bash
+$ curl -i $PROXY_IP/get
+HTTP/1.1 200 OK
+Content-Type: text/html; charset=utf-8
+Content-Length: 0
+Connection: keep-alive
+Server: gunicorn/19.9.0
+Date: Wed, 17 Jul 2019 19:25:32 GMT
+Access-Control-Allow-Origin: *
+Access-Control-Allow-Credentials: true
+X-Kong-Upstream-Latency: 2
+X-Kong-Proxy-Latency: 1
+Via: kong/1.2.1
+
+$ curl -i --data "foo=bar" -X POST $PROXY_IP/post
+HTTP/1.1 200 OK
+Content-Type: text/html; charset=utf-8
+Content-Length: 0
+Connection: keep-alive
+Server: gunicorn/19.9.0
+Date: Wed, 17 Jul 2019 19:25:32 GMT
+Access-Control-Allow-Origin: *
+Access-Control-Allow-Credentials: true
+X-Kong-Upstream-Latency: 2
+X-Kong-Proxy-Latency: 1
+Via: kong/1.2.1
+
+{
+  "args": {},
+  "data": "",
+  "files": {},
+  "form": {
+    "foo": "bar"
+  },
+  "headers": {
+    "Accept": "*/*",
+    "Connection": "keep-alive",
+    "Content-Length": "7",
+    "Content-Type": "application/x-www-form-urlencoded",
+    "Host": "localhost",
+    "User-Agent": "curl/7.54.0",
+    "X-Credential-Identifier": "localhost",
+    "X-Forwarded-Host": "localhost"
+  },
+  "json": null,
+  "origin": "192.168.0.3",
+  "url": "http://some.url/post"
+}
+
+```
+
+## Add JWT authentication to the service
+
+With Kong, adding authentication in front of an API is as simple as
+enabling a plugin. Let's enable JWT authentication
+
+```bash
+$ echo "
+apiVersion: configuration.konghq.com/v1
+kind: KongPlugin
+metadata:
+  name: app-jwt
+plugin: jwt
+" | kubectl apply -f -
+```
+
+Now let's associate the plugin to the Ingress rules we created earlier.
+
+```bash
+$ echo '
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: demo-get
+  annotations:
+    plugins.konghq.com: app-jwt
+    konghq.com/strip-path: "false"
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /get
+        backend:
+          serviceName: httpbin
+          servicePort: 80
+' | kubectl apply -f -
+
+$ echo '
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: demo-post
+  annotations:
+    plugins.konghq.com: app-jwt
+    konghq.com/strip-path: "false"
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /post
+        backend:
+          serviceName: httpbin
+          servicePort: 80
+' | kubectl apply -f -
+```
+
+Any requests matching the proxying rules for `demo-get` and `demo` post will now require a valid JWT and the consumer for the JWT to be associate with the right ACL.
```

Comment:
Can you update the entire doc to respect the 80 char column limit? Please wrap around long sentences.

Reply:
done